src/i18n/de.js

```diff
@@ -101,7 +102,9 @@ export default {
   resendLabel: 'Erneut senden',
   resendingLabel: 'Wird erneut gesendet...',
   retryLabel: 'Wiederholen',
-  sentLabel: 'Senden',
+  sentLabel: 'Gesendet!',
+  showPassword: 'Passwort anzeigen'
```

Comment:
you're missing a `,` at the end of the line:

```suggestion
  showPassword: 'Passwort anzeigen',
```

Reply:
thank you for catching that. This used to be the last line of the document, which didn't end in a comma but I moved it up to have the same structure as the English version.